teste2e/author_submit_extrinsic_test.go

```diff
@@ -52,15 +53,15 @@ func TestChain_SubmitExtrinsic(t *testing.T) {
 		panic(err)
 	}
 
-	c, err := types.NewCall(meta, "Balances.transfer", bob, types.UCompact(6969))
+	// NOTE: for chains with out pallet_indices, use the following instead set SkipAddressPrefix to true
+	ctx := context.WithValue(context.Background(), types.SkipAddressPrefix, false)
```

Comment:
@philipstanislaus this is the main option flag. It needs to be used for all the subsequent calls.